Dalamud/Game/Gui/PartyFinder/PartyFinderGui.cs

```diff
@@ -38,22 +42,11 @@ private PartyFinderGui(SigScanner sigScanner)
         this.receiveListingHook = Hook<ReceiveListingDelegate>.FromAddress(this.address.ReceiveListing, new ReceiveListingDelegate(this.HandleReceiveListingDetour));
     }
 
-    /// <summary>
-    /// Event type fired each time the game receives an individual Party Finder listing.
-    /// Cannot modify listings but can hide them.
-    /// </summary>
-    /// <param name="listing">The listings received.</param>
-    /// <param name="args">Additional arguments passed by the game.</param>
-    public delegate void PartyFinderListingEventDelegate(PartyFinderListing listing, PartyFinderListingEventArgs args);
-
     [UnmanagedFunctionPointer(CallingConvention.ThisCall)]
     private delegate void ReceiveListingDelegate(IntPtr managerPtr, IntPtr data);
 
-    /// <summary>
-    /// Event fired each time the game receives an individual Party Finder listing.
-    /// Cannot modify listings but can hide them.
-    /// </summary>
-    public event PartyFinderListingEventDelegate ReceiveListing;
+    /// <inheritdoc/>
+    public event IPartyFinderGui.PartyFinderListingEventDelegate ReceiveListing;
```

Comment:
This is a simple enough service that I'm going to ask that it be made scoped now.